src/controllers/userController.js

```diff
@@ -50,6 +73,17 @@ const showCurrentUser = async (req, res) => {
 
 // Update the information of the current user
 const updateCurrentUser = async (req, res) => {
+  /*
+     #swagger.summary = 'Fetch a user by id, update their data and return a new token user.'
```

Comment:
```suggestion
     #swagger.summary = 'Fetch a user by id, update their data and return a new user.'
```

We just return a regular full (except password) user; not the tokenized one that we put in the cookie